sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingDeduplicationSuite.scala

```diff
@@ -574,6 +574,24 @@ class StreamingDeduplicationSuite extends StateStoreMetricsTest {
       matchPVals = true
     )
   }
+
+  test("test that avro encoding is not supported") {
```

Comment:
Can we add a test case that uses TWS and another stateful operator?

Reply:
Done